src/main.cpp

```diff
@@ -14,28 +14,42 @@
 // Sensor
 #include "sensor.hpp"
 
+RTC_DATA_ATTR static int lastValue = -1; // remember number in RTC Memory
+
+#define INTERVAL_TIME 10
+
 // Define Firebase variables
 #define WIFI_SSID wifi_ssid
 #define WIFI_PASSWORD wifi_password
 
-unsigned long sendDataPrevMillis = 0;
-unsigned long count = 0;
-
 // Define SheetDB object
 const char *url = host_url;
 SheetDB sheetDB;
 StaticJsonDocument<JSON_OBJECT_SIZE(request_key)> json_request;
 StaticJsonDocument<JSON_OBJECT_SIZE(response_key)> json_response;
-
 // Define FirebaseRTDBClient object
 FirebaseRTDBClient firebaseRTDBClient;
-
+FirebaseJson json;
 // Define Sensor
 Sensor sensor;
 
+int value;
+
 void setup()
 {
-  M5.begin();
+  setCpuFrequencyMhz(80);
+  M5.begin(false, true, true);
+  M5.Axp.ScreenBreath(0);
+  // Initialize Sensor
+  sensor.setPin(36);
+  value = sensor.getValue();
+  if (value == lastValue) //  センサの値が前回更新した値と等しいとき
```

Comment:
```suggestion
  if (value == lastValue && DEBUG_MODE == false) //  センサの値が前回更新した値と等しいとき
```
確かに重要ではあるんですが、デバッグの関係上データをずっと上げるようにして欲しさあります
なので `DEBUG_MODE` を定義してそこで管理しましょう